src/components/big-interactive-pages/editor.tsx

```diff
@@ -71,13 +72,36 @@ const saveGame = debounce(800, (persistenceState: Signal<PersistenceState>, code
 	lastSavePromise = (lastSavePromise ?? Promise.resolve()).then(doSave)
 })
 
+const exitTutorial = (persistenceState: Signal<PersistenceState>) => {
```

Comment:
My thesis is that this should work in non-persisted states, simply removing the tutorial in-memory. What exactly are you unsure about in implementing that?

Reply:
I originally thought none of the changes were saved in-memory and would be reset on refresh (based on how the ui describes it), I see how it'd be useful now. I'll attempt an implementation.

Reply:
I don't think I fully understood the question. From what I understood originally. The goal was to be able to remove the tutorial from non `PERSISTED` games? If so, tutorials can only exist on gallery (`SHARED`) or `PERSISTED` games. Gallery games cannot be saved without turning it into a `PERSISTED` game (through remix).

Alternatively, if you are talking about not using `PersistenceState` to store the tutorial. I used `PersistenceState` as it seemed to be the main thing passing data between editor components. It is possible to pass it around as a prop, but reactivity seems to be somewhat broken/complicated to fix when it is removed.